curation_concerns-models/app/jobs/ingest_local_file_job.rb

```diff
@@ -22,16 +22,18 @@ def run
 
     if actor.create_content(File.open(path), filename, mime_type(filename))
       FileUtils.rm(path)
-      CurationConcerns.queue.push(ContentDepositEventJob.new(generic_file.id, user_key))
 
-      message = "The file (#{File.basename(filename)}) was successfully deposited."
-      subject = 'Local file ingest'
```

Comment:
We don't want to lose this code, I imagine, and we'll want `subject` sent to the method call on line 28.